src/waveform/renderers/allshader/waveformrendererstem.h

```diff
@@ -18,30 +15,31 @@ class WaveformRendererStem;
 
 class allshader::WaveformRendererStem final
         : public allshader::WaveformRendererSignalBase,
-          public rendergraph::OpenGLNode {
+          public rendergraph::GeometryNode {
   public:
     explicit WaveformRendererStem(WaveformWidgetRenderer* waveformWidget,
             ::WaveformRendererAbstract::PositionSource type =
                     ::WaveformRendererAbstract::Play);
 
-    // override ::WaveformRendererSignalBase
+    // Pure virtual from WaveformRendererSignalBase, not used
     void onSetup(const QDomNode& node) override;
 
-    void initializeGL() override;
-    void paintGL() override;
+    bool init() override;
 
-  private:
-    mixxx::RGBAShader m_shader;
-    mixxx::TextureShader m_textureShader;
-    VertexData m_vertices;
-    RGBAData m_colors;
+    bool supportsSlip() const override {
+        return true;
```

Comment:
After testing, looks like the stem waveform doesn't support the slip animation

Reply:
not sure what you mean by this. are you saying this should this return false? or that there is a bug?

Reply:
Actually I found an unrelated bug! If you have `High Details` enabled, it the slip animation doesn't work! Not introduced by your changes tho!

Reply:
Ok, so I can ignore this? Please resolve the conversation if so.